config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "academy": "",
+        "readthedocs": "knowage",
+        "helpdesk": "Knowage",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "KnowageLabs/Knowage-Documentation",
+            "KnowageLabs/Knowage-Server",
+            "KnowageLabs/Knowage-Server-Chef",
+            "KnowageLabs/Knowage-Server-Docker"
+        ],
+        "docker": [
+            "knowagelabs/knowage-server-docker"
+        ]
+    },
+    {
+        "enabler": "Kurento",
+        "catalogue": "kurento",
+        "academy": "Kurento",
+        "readthedocs": "doc-kurento",
+        "helpdesk": "Kurento",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Kurento/doc-fiware-readthedocs",
+            "Kurento/kurento-media-server",
+            "Kurento/kurento-docker"
+        ],
+        "docker": [
+            "fiware/stream-oriented-kurento"
+        ]
+    },
+    {
+        "enabler": "MR CoAP - Protocol Adapter",
+        "catalogue": "protocol-adapter-mr-coap",
+        "academy": "Protocol Adapter",
+        "readthedocs": "",
+        "helpdesk": "",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": []
+    },
+    {
+        "enabler": "Micro-XRCE-DDS",
+        "catalogue": "fast-rtps",
+        "academy": "",
+        "readthedocs": "",
+        "helpdesk": "Micro-XRCE-DDS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "eProsima/Micro-XRCE-DDS"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Murano",
+        "catalogue": "application-management-murano",
+        "academy": "Murano",
+        "readthedocs": "murano",
+        "helpdesk": "Murano",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": [
+            "fiware/murano"
+        ]
+    },
+    {
+        "enabler": "OFNIC - Network Information & Control",
+        "catalogue": "network-information-and-control-ofnic",
+        "academy": "OFNIC",
+        "readthedocs": "fiware-ofnic",
+        "helpdesk": "OFNIC",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "FIWARE-UNIROMA1/FIWARE-OFNIC"
+        ],
+        "docker": [
+            "fiware/ofnic"
+        ]
+    },
+    {
+        "enabler": "Object Storage",
+        "catalogue": "object-storage-ge-fiware-implementation",
+        "academy": "Object Storage",
+        "readthedocs": "",
+        "helpdesk": "Object Storage",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "coverll": "renarsvilnis/fiware-object-storage-ge",
+        "github": [
+            "renarsvilnis/fiware-object-storage-ge"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "OpenMTC - oneM2M",
```

Comment:
"enabler": "OpenMTC"

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585